lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -0,0 +1,91 @@
+{{#if @manager.inEditMode}}
+    <ValidatedModelForm
```

Comment:
Hmm, the manager pattern doesn't mesh well with `ValidatedModelForm`, since VMF itself manages saving changes. We should probably break it into pieces, like a "smart" `ValidatedChangeManager` and a collection of "dumb" rendering components.

Talked in person -- this approach is fine for now, but we should be thinking about how our form handling can be improved.